Test failure: `com.linecorp.armeria.client.zookeeper.ServerSetDiscoveryTest.initializationError`

```
java.lang.RuntimeException: Failed to start a ZooKeeper instance
	at com.linecorp.armeria.common.zookeeper.ZooKeeperExtension.before(ZooKeeperExtension.java:48)
	at com.linecorp.armeria.testing.junit5.common.AbstractAllOrEachExtension.beforeAll(AbstractAllOrEachExtension.java:56)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
Caused by: java.util.concurrent.TimeoutException: Timeout waiting for Future to complete
	at javascalautils.concurrent.FutureImpl.ready(FutureImpl.java:245)
	at javascalautils.concurrent.FutureImpl.result(FutureImpl.java:228)
	at javascalautils.concurrent.Future.result(Future.java:395)
	at com.linecorp.armeria.common.zookeeper.ZooKeeperExtension.before(ZooKeeperExtension.java:46)
	... 2 more

```

--- a/zookeeper3/src/test/java/com/linecorp/armeria/common/zookeeper/ZooKeeperExtension.java
+++ b/zookeeper3/src/test/java/com/linecorp/armeria/common/zookeeper/ZooKeeperExtension.java
@@ -33,7 +33,7 @@ import zookeeperjunit.ZooKeeperAssert;
 
 public class ZooKeeperExtension extends AbstractAllOrEachExtension implements ZooKeeperAssert {
 
-    private static final Duration duration = Duration.ofSeconds(10);
+    private static final Duration duration = Duration.ofSeconds(60);
     private static final TemporaryFolder tempFolder = new TemporaryFolder();
 
     @Nullable

